khmer/kfile.py

```diff
@@ -151,7 +151,12 @@ def check_space(in_files, force, _testhook_free_space=None):
 
 def check_space_for_graph(outfile_name, hash_size, force,
                           _testhook_free_space=None):
-    """Check that we have enough size to write the specified graph."""
+    """
+    Check that we have enough size to write the specified graph.
+
+    The "hash_size" parameter should equal the total bytes required for the
+    entire data structure.
+    """
     dir_path = os.path.dirname(os.path.realpath(outfile_name))
     target = os.statvfs(dir_path)
```

Comment:
By the way: `statvfs` is deprecated. Not urgent for this PR, but something to be aware of. https://docs.python.org/2/library/statvfs.html